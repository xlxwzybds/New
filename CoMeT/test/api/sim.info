{'cmdline': ['../../run-sniper',
             '-n',
             '2',
             '-c',
             'gainestown',
             '--roi',
             '-g',
             '--api/simple/cores_per_socket=1',
             '--',
             './api'],
 'git_revision': '527b99e06f8fc217cf4b2616a7151f0114c7c915',
 'host': 'shailja-OptiPlex-7050',
 'pin_version': 'unknown',
 'rusage': (0.137977,
            0.020905,
            29892,
            0,
            0,
            0,
            5840,
            0,
            0,
            0,
            144,
            0,
            0,
            0,
            615,
            0),
 'snipercmd': ['bash',
               '-c',
               '/home/shailja/workspace/research_2020/sniper/lib/sniper -c /home/shailja/workspace/research_2020/sniper/config/base.cfg --general/total_cores=2 --general/output_dir=/home/shailja/workspace/research_2020/sniper/test/api --config=/home/shailja/workspace/research_2020/sniper/config/nehalem.cfg --config=/home/shailja/workspace/research_2020/sniper/config/gainestown.cfg --api/simple/cores_per_socket=1 -g --general/magic=true -g --traceinput/stop_with_first_app=true -g --traceinput/restart_apps=false -g --traceinput/stop_with_first_app=false -g --traceinput/enabled=true -g --traceinput/emulate_syscalls=true -g --traceinput/num_apps=1 -g --traceinput/trace_prefix=/tmp/tmpXfzONE/run_benchmarks'],
 't_elapsed': 2.5866081714630127,
 't_start': 1599473894.813223,
 'tracecmds': [['/home/shailja/workspace/research_2020/sniper/record-trace',
                '-o',
                '/tmp/tmpXfzONE/run_benchmarks',
                '--roi',
                '-e',
                '1',
                '-s',
                '0',
                '-r',
                '1',
                '--follow',
                '--routine-tracing',
                '--',
                './api']],
 'user': 'shailja',
 'vmem': 42516480L}
