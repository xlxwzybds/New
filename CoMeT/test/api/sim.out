                                     | Core 0     | Core 1    
  Instructions                       |      72594 |          0
  Cycles                             |     113359 |     113359
  IPC                                |       0.64 |       0.00
  Time (ns)                          |      42617 |      42617
  Idle time (ns)                     |      10487 |      42617
  Idle time (%)                      |      24.6% |     100.0%
Branch predictor stats               |            |           
  num correct                        |      10476 |          0
  num incorrect                      |         53 |          0
  misprediction rate                 |      0.50% |      0.00%
  mpki                               |       0.73 |       0.00
TLB Summary                          |            |           
  I-TLB                              |            |           
    num accesses                     |      20329 |          0
    num misses                       |         13 |          0
    miss rate                        |      0.06% |      0.00%
    mpki                             |       0.18 |       0.00
  D-TLB                              |            |           
    num accesses                     |      61007 |          0
    num misses                       |          9 |          0
    miss rate                        |      0.01% |      0.00%
    mpki                             |       0.12 |       0.00
  L2 TLB                             |            |           
    num accesses                     |         22 |          0
    num misses                       |         22 |          0
    miss rate                        |    100.00% |      0.00%
    mpki                             |       0.30 |       0.00
Cache Summary                        |            |           
  Cache L1-I                         |            |           
    num cache accesses               |      20329 |          0
    num cache misses                 |         84 |          0
    miss rate                        |      0.41% |       inf%
    mpki                             |       1.16 |        inf
  Cache L1-D                         |            |           
    num cache accesses               |      61007 |          0
    num cache misses                 |         38 |          0
    miss rate                        |      0.06% |       inf%
    mpki                             |       0.52 |        inf
  Cache L2                           |            |           
    num cache accesses               |        130 |          0
    num cache misses                 |        122 |          0
    miss rate                        |     93.85% |       inf%
    mpki                             |       1.68 |        inf
  Cache L3                           |            |           
    num cache accesses               |        130 |          0
    num cache misses                 |        122 |          0
    miss rate                        |     93.85% |       inf%
    mpki                             |       1.68 |        inf
DRAM summary                         |            |           
  num dram accesses                  |        122 |          0
  average dram access latency (ns)   |     109.03 |        inf
  average dram queueing delay        |      55.60 |       0.00
  average dram bandwidth utilization |      2.00% |      0.00%
Coherency Traffic                    |            |           
  num loads from dram                |         26 |          0
  num loads from dram cache          |          0 |          0
  num loads from remote cache        |          0 |          0
